src/Lib/LanguagesLib.php

```diff
@@ -543,6 +543,7 @@ public static function languagesInTatoeba()
                 'urh' => __d('languages', 'Urhobo'),
                 'mvv' => __d('languages', 'Tagal Murut'),
                 'mdf' => __d('languages', 'Moksha'),
+                'mdr' => __d('languages', 'Mandar'),
```

Comment:
Can you please add the language at the very end of the list, instead of the middle? This is what we have been doing so far and I don’t want to unnecessarily change this habit.

Reply:
okay